src/Zynga/Framework/IO/Web/V1/Curl/CurlInterface.hh

```diff
@@ -0,0 +1,10 @@
+<?hh // strict
+
+namespace Zynga\Framework\IO\Web\V1\Curl;
+
+interface CurlInterface {
+  public function setOptionsArray(array<int, mixed> $options): bool;
+  public function execute(): mixed;
```

Comment:
Why is the return type mixed? We should try to not allow mixed wherever we can because it allows the bad PHP practice of juggling return types and lack of contracrts.